src/ResourceManager/HDInsight/Commands.HDInsight.Test/UnitTests/DataLakeStoreTests.cs

```diff
@@ -56,7 +56,7 @@ public DataLakeStoreTests()
             };
         }
 
-        [Fact]
+        //[Fact]
```

Comment:
This test was temporarily removed since it has been failing for 3 months. The owner of this test agreed to fix it in the next release.